### ISSUE TITLE
Discord Avatar Bug

### ISSUE THREAD

**Issue by Arinji2** [rafaelalmeidatk/nextjs-forum#89]
![image](https://github.com/user-attachments/assets/20c546b5-3433-4da2-9f5d-a86d433aa145)
Just noticed this bug for Fuma, not sure why the links are seemingly "broken"

[Fuma Image](https://cdn.discordapp.com/avatars/572329183334891520/3ea2e2e134a2327e2b8a7738876a17b5.webp)

[Yi Lon Ma Image](https://cdn.discordapp.com/guilds/752553802359505017/users/510480545160101898/avatars/182bf1e5d47028fa0f075f7acce2a045.webp)

I think its something related to the /avatars and /guilds

Will investigate and see if i can fix this

**Comment by anay-208** [comment]
# Cause
As discussed in [#discord-feedback](https://discord.com/channels/752553802359505017/843997308616966145), this error was caused due to the fact that the user got his profile pic updated, so the Old Url was no longer active

Profile Pic gets updated, if the user is active in help-forum

This is not only with fuma nama, but if you view some old posts like one here: https://nextjs-forum.com/page/1201

# Solution
A suitable solution for the issue is to use [Radix Avatar](<https://www.radix-ui.com/primitives/docs/components/avatar>), it has a fallback option, which can be used to display a default profile pic in frontend.

**Comment by anay-208** [comment]
@rafaelalmeidatk What do you think about the solution?

**Comment by rafaelalmeidatk** [comment]
I don't think we need to install a radix component just for that, images fallbacks are very simple to implement

**Comment by Arinji2** [comment]
@rafaelalmeidatk I could just do a onError check on the image, and render a default discord avatar? If that sounds fine let me know and I can make a quick pr for the same.

**Comment by rafaelalmeidatk** [comment]
Yeah that sounds good

**Comment by Arinji2** [comment]
Noted, will create one once I get the time. 👍🏻
